### PR TITLE
fix: guard against exceptions from canboatjs

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: packages/streams/canboatjs.js [hunk 1]
```diff
@@ -48,18 +48,22 @@ function CanboatJs (options) {
...
       }
-    })
-  } else {
-    this.fromPgn.parse(chunk, this.handlePgnData)
+      })
+    } else {
+      this.fromPgn.parse(chunk, this.handlePgnData)
+    }
+  } catch ( error ) {
+    console.error(err)
```
Comment (tkurki) [inline packages/streams/canboatjs.js:66]: This will fail when failing...err vs error. TypeScript, anyone..

Comment (sbender9) [reply packages/streams/canboatjs.js:66]: ouch

Comment (sbender9) [reply packages/streams/canboatjs.js:66]: nice catch!

Comment (sbender9) [reply packages/streams/canboatjs.js:66]: fixed and actually tested the error condition this time

**Pull request merged**